### DISCUSSION TITLE
Expanding Hall of Fame (High Scores)

### DISCUSSION THREAD

**Discussion by Darkloke** [ihhub/fheroes2#9181] [Ideas]
Greetings. Right now High Score contains top ten results for completed maps. What do you think about "expanding" it by adding scrolling there and basically turning it to "victory log", where you also could possibly see the date, when you completed the map?

Another question. Lets say you completed the map and had your high score set. What will happen if you reload the earlier save and continue to play by prolonging the game - when you beat it - will you have another HS entry or it will rewrite existing for the map? May be it would be possible to create a some kind of GUID for each map session, so such victories would update your HS, but not create another one?

**Comment by zenseii** [comment]
Greetings back, @Darkloke !

We have discussed this and expanding the size of the High Scores is something that would be very nice but we've had to prioritize other things. Reworking the info supplied in high scores like date achieved is also possible and I'm sure others would like this too. I suppose the best way to have this extra info is to be able to right-click each score instead of trying to fit more info into the already constrained UI.

As for your point about continuing to play a session and ending up with two scores for a game, I believe your suggestion would become very complex very quickly. I think an easier solution would be to be able to delete manually high score entries so that you can avoid duplicates.

**Comment by fheroes2bugs** [comment]
I already created issue about high scores:

https://github.com/ihhub/fheroes2/discussions/7917
